### PR TITLE
Add pages read 73

### PR DESCRIPTION
**Pull request by YBrecher**
## Summary of change

Added a pagesRead field that will only be displayed in the "Did Not Finish Shelf". Also updated the relevant tests.

## Additional context

Screenshot (64)

## Related issue

https://github.com/knjk04/book-project/issues/73

### CODE REVIEW

**Comment by knjk04** [comment]
Thanks for this. Before I review this pull request, could you resolve the merge conflicts?

**Comment by YBrecher** [comment]
Sure, no problem

File: src/test/java/com/karankumar/bookproject/ui/book/BookFormTests.java [hunk 1]
```diff
@@ -214,24 +223,31 @@ public void correctFormFieldsShownForGivenShelf(PredefinedShelf.ShelfName shelfN
...
             case DID_NOT_FINISH:
                 shouldShowStarted = true;
+                shouldShowPagesRead = true;
                 // finished and rating already set to false
                 break;
             case READ:
                 shouldShowStarted = true;
                 shouldShowFinished = true;
                 shouldShowRating = true;
+                shouldShowPagesRead= false;
```
Comment (knjk04) [inline src/test/java/com/karankumar/bookproject/ui/book/BookFormTests.java:243]: could you add a space before the equals sign?

**Comment by YBrecher** [comment]
Sure no problem

**Comment by knjk04** [comment]
Thanks again!

**Pull request merged**